src/app/features/home/home.page.scss

```diff
@@ -11,14 +11,39 @@ mat-sidenav {
   width: 60%;
 }
 
+mat-tab {
+  height: 100%;
+  background-color: brown;
+  display: flex;
+}
+
+mat-tab-bady {
+  height: 100%;
+  background-color: yellow;
+  display: flex;
+}
+
 mat-sidenav-content {
   width: 100%;
   overflow-y: scroll;
   overflow-x: hidden;
 }
 
-.capture-fab {
-  position: fixed;
-  bottom: 16px;
-  right: 16px;
+.camera-tab-button {
+  height: 48px;
+  width: 48px;
+}
+
+.camera-tab-icon {
+  height: 24px;
+  width: 24px;
+  color: white;
+}
+
+#tab-group {
+  height: calc(100% - 90px);
```

Comment:
This hardcoded height will cause the white space between bottom navigation and the bottom of the screen.

![Screenshot from 2021-01-28 11-27-11](https://user-images.githubusercontent.com/14951000/106085977-2e80a380-615c-11eb-87f7-1412798ab348.png)